packages/workflows/__tests__/__unit__/Create.unit.test.ts

```diff
@@ -592,7 +598,7 @@ describe("Create workflow from local file", () => {
             let response: any;
             try {
                 response = await CreateWorkflow.createWorkflowLocal(PRETEND_SESSION, wfName, wfDefinitionFile, systemName, wfOwner,  null,
-                    null, null, null, null, false, null, "");
+                    null, null, null, null, false, null, null, "");
```

Comment:
Should we create an interface that substitutes all 10+ parameters going into the `createWorkflow` API?
Also, we should probably deprecate it instead of breaking it right away. 😋

Reply:
Oh, that would be so much better :D.